File: TS/ex00/multiples_of_3_or_5.ts
function	multiples_of_3_or_5(n: number): number
{
	let	i : number = 0;
	let	sum : number = 0;

	while (i < n)
	{
		if ((i%3 == 0) || (i%5 == 0))
			sum += i;
		i++;
	}
	return sum;
}

console.log(multiples_of_3_or_5(10));

File: PYTHON/ex00/multiples_of_3_or_5.py
def multiples_of_3_or_5(n):
    i = 0
    sum = 0
    while (i < n):
        if ((i%3 == 0) or (i%5 == 0)):
            sum += i
        i += 1;
    return sum

print(multiples_of_3_or_5(10))
